Dockerfile

```diff
@@ -125,7 +125,7 @@ RUN apt update && apt-get install -y libbz2-1.0 \
         libxml2 \
         zlib1g \
         tzdata \
-        netcat && \
+        netcat-traditional && \
```

Comment:
I think the direct replacement for what we had before is `netcat-openbsd` but it probably doesn't matter much 🤷